configure.ac

```diff
@@ -558,6 +558,7 @@ src/Makefile
 src/cli/Makefile
 src/ncp/Makefile
 src/core/Makefile
+src/diag/Makefile
```

Comment:
@mszczodrak Yes, the latest commit is to address your comments, but it failed to pass all the checks. I will put another patch to address the check errors, as well as the comments from @jwhui . Thanks.